python-scripts/xbox/interface/if_xbdm.py

```diff
@@ -190,11 +205,15 @@ def write1(address, data, physical):
 
   from xbox.pe import *
 
-  modules = GetModules()
-  # FIXME: Get location of xbdm.dll
-  xbdm_base = 0x0B0011000
-  DmResumeThread_addr = resolve_export(35, image_base=xbdm_base) #FIXME: Use location of xbdm.dll
-  #DmResumeThread_addr = resolve_export(35) #FIXME: Use location of xbdm.dll
+  modules=GetModules() #init modules
```

Comment:
this does not "init" anything. The function name itself should also be descriptive enough.
= Remove comment.

(Also spaces before and after `=` would be appreciated)